author/topics/document-format/requirements-recommendations-permissions.adoc

```diff
@@ -48,7 +49,7 @@ presupposed by this requirement.
 [source,asciidoc]
 .An exhaustive attribute example
 -----
-[.recommendation,label="/ogc/recommendation/wfs/2",subject="user",inherit="/ss/584/2015/level/1",classification="control-class:Technical;priority:P0;family:System and Communications Protection,System and Communications Protocols",obligation="permission,recommendation",model=ogc,type=verification]
+[.recommendation,label="/ogc/recommendation/wfs/2",subject="user",inherit="/ss/584/2015/level/1;ISO 693-2 Clause 4.1",classification="control-class:Technical;priority:P0;family:System and Communications Protection,System and Communications Protocols",obligation="permission,recommendation",model=ogc,type=verification]
```

Comment:
This example is quite strange. Why are we adding `ISO 693-2`?

We should make use a real example from an OGC document.

Reply:
Only if we have a real example that includes ALL attributes, which is the point of this example. 

The example is meant to look weird; it illustrates that inherits can come from documents as well as other recommendations (an expansion OGC themselves requested).